components/eam/src/chemistry/modal_aero/aero_model.F90

```diff
@@ -2834,6 +2848,18 @@ subroutine aero_model_emissions( state, cam_in )
        sflx(:)=0._r8
        do m=1,dust_nbin+dust_nnum
           mm = dust_indices(m)
+!<shanyp 07052024
+          if(dstemislimitswitch) then
+           do icol=1,ncol
+            if((cam_in%cflx(icol,mm).ge.dstemislimit).and.(m.le.dust_nbin)) then
+             mmn=dust_indices(m+2)
+             cam_in%cflx(icol,mmn)=cam_in%cflx(icol,mmn)*dstemislimit/cam_in%cflx(icol,mm)
+             cam_in%cflx(icol,mm)=dstemislimit
+             write(iulog,'(a,1x,i10)') 'The dust emission cap is hit at icol=', icol
```

Comment:
I've found that the `icol` value typically is not helpful in identifying the geographic location where things occur (are triggered). Would it be possible to get the `lat` `lon` values and also the timestep value so that we know where and when they occur?